x-pack/test_serverless/functional/test_suites/search/elasticsearch_start.ts

```diff
@@ -153,7 +153,6 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await pageObjects.svlSearchElasticsearchStartPage.clickCreateIndexButton();
         await pageObjects.svlSearchElasticsearchStartPage.expectToBeOnIndexDetailsPage();
 
-        await pageObjects.svlApiKeys.expectAPIKeyAvailable();
```

Comment:
This expect does the check to see if the apiKey is visible and retries (deletes api keys) when not available. I suggest this needs to be in otherwise the tests will be flakey.

Reply:
The thing is when we call `expectAPIKeyAvailable` before, it set the API key visibility to true, and second time when we call this function it checked again on the MASK, but we have previous state as visible, so we need to check if key is visible. I can create split `expectAPIKeyAvailable` in two functions, and call that one which check already visible key

Reply:
expectAPIKeyAvailable should be changed only to evaluate whether the api key input is present, not whether its masked or not. Its there to make sure we have some sort of API key present and not the action to create a new API key.